### ISSUE TITLE
Global Bootstrap color style overrides for AA contrast accessibility 

### ISSUE THREAD

**Issue by pbastia** [bcgov/cas-ciip-portal#1810]
There are Bootstrap color variants (with resulting background-color, border-color, and color styles) throughout the site that always need to be globally overridden to meet WCAG AA contrast. We should do these overrides once in the default layout, instead of haphazardly throughout the code base.

Ideally, we can then remove the inline overrides scattered everywhere.

Focus:

The issue comes mainly from Button and Badge components.
Implement overrides for the following non-accessible variants of both Button and Badge:
```
        .btn-primary
        .btn-success
        .btn-info
        .btn-link
        .btn-outline-warning (the regular .btn-outline is fine as-is)
````
I'm not sure what would be the point of implementing .btn-outline-light as it's practically invisible...
Ensure each of the related outline- variants are overridden with the same colors and pass AA contrast
Ensure all variants' :focus, :active, :hover, :disabled and .disabled states are also overridden with the accessible colors.

See discussion in GGIRCS-2405

**Comment by pbastia** [comment]
[...] first clean up, prune and organize the styles in the default layout: that's the finished result of what you'll see in the [PR #1741](https://github.com/bcgov/cas-ciip-portal/pull/1741)

**Comment by pbastia** [comment]
@kriscooke wrote:
Next up is the more straightforward part of moving contrast color-related styles into the default layout. I created a dedicated style module which is imported there, `GlobalBootstrapOverrideStyles` where these should be added:

`GenericConfirmationModal` has a good example of overriding the 'primary' button color background (no need to be id-specific), including transient states like hover. All of these should be moved into the above style module. Note the several pseudoselector styles to be overridden on `:hover`, `:focus`, `:active`. I believe there may be another one BS uses sometimes for `:not([disabled])`, `:not(.disabled)`.

The best way to unify these contrast overrides is to go on the Bootstrap component documentation site and inspect the offending elements: Buttons (regular and outline/inverted) and Badges.

- Observe the outline/background styles applied, note the selector specified by Bootstrap and use the exact same selectors in the module to override successfully.
- Use the devtools to toggle states like `:hover`, `:focus`, `:active` and `[disabled]` and repeat the above.

We've already chosen contrast-compliant colors in various parts of the site but there are some missing selectors, like the transient states mentioned above.

Once all the color overrides are specified, it would be ideal to remove the one-offs located around the site to ensure uniformity.
